src/iterative_ensemble_smoother/__init__.py

```diff
@@ -138,6 +139,10 @@ def update_step(
             step_length = self._get_steplength(self._module_data.iteration_nr)
         if noise is None:
             noise = np.random.rand(*S.shape)
+        if ensemble_mask is None:
```

Comment:
This change should probably be a separate commit and not end up in "Update polynomial notebook with new public interface".

Reply:
Done now